test/apps/test_ucx_tls.py

```diff
@@ -140,6 +169,59 @@ def test_fallback_from_rc(dev, neps) :
 
     os.unsetenv("UCX_TLS")
 
+def test_ucx_tls_positive(tls):
+    # Use TLS list in "allow" mode and verify that the found tl is in the list
+    found_tl = find_am_transport(None, tls=tls)
+    print "Using UCX_TLS=" + tls + ", found TL: " + str(found_tl)
+    if tls == 'all':
+        return
+    tls = tls.split(',')
+    if found_tl in tls:
+        return
+    for tl in tls:
+        if tl in tl_aliases:
+            if found_tl in tl_aliases[tl]:
+                return
+    sys.exit(1)
```

Comment:
pls print a detailed error

Reply:
Added (originally I put only `sys.exit(1)` because that's what I saw in other places here).